x-pack/plugins/cases/server/connectors/thehive/format.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Format } from './types';
+import type { ConnectorTheHiveTypeFields} from '../../../common/types/domain';
+
+function mapSeverity(severity: string): number{
+  switch (severity) {
+    case 'low':
+      return 1;
+    case 'medium':
+      return 2;
+    case 'high':
+      return 3;
+    case 'critical':
+      return 4;
+    default:
+      return 2;
+  }
+}
+
+export const format: Format = (theCase) => {
+  const {
+    tlp = null
+  } = (theCase.connector.fields as ConnectorTheHiveTypeFields['fields']) ?? {};
+  return {
+    tags: theCase.tags,
+    tlp: typeof tlp === 'string' ? parseInt(tlp) : tlp,
```

Comment:
It seems we have a rule for the radix parameter of the `parseInt` call, the linter is complaining about it. To fix it:
```suggestion
    tlp: typeof tlp === 'string' ? parseInt(tlp, 10) : tlp,
```